packages/zapp/console/src/models/Common/constants.ts

```diff
@@ -1,9 +1,13 @@
 import axios, { AxiosRequestConfig, AxiosTransformer } from 'axios';
 import * as camelcaseKeys from 'camelcase-keys';
 import * as snakecaseKeys from 'snakecase-keys';
-import { isObject } from 'util';
 import { LiteralMapBlob, ResourceType, SystemStatus } from './types';
 
+// recommended util.d.ts implementation
```

Comment:
nit: yeah this should put in util

Reply:
Noted: Will put it in util, but not in this PR :)
thanks